pyconcz/common/models.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+
+
+class SiteFlag(models.Model):
+    name = models.CharField(max_length=200, unique=True)
+    value_type = models.CharField(
+        max_length=200,
+        choices=[
+            # Human-readable titles are the same as the value
+            (v, v) for v in ("on", "off", "on_at", "off_at")
```

Comment:
```suggestion
            (v, v) for v in ("on", "off", "on_at", "off_at")
```
ty choices bejva fajn mit jako "konstantu" na ty tride (kdyz se to meni, tak se to meni na jednom miste, pouziva se to pak z klasy, je to takovy vic jasny, co to dela), ale nevidim to jako velkej trabl....

neco ala:

VALUE_TYPE_ON = 'on',  nebo se na to pouzivaj libky ala `django-choices`